src/components/Footer.tsx

```diff
@@ -0,0 +1,52 @@
+import { FilterType } from '../types/FilterType';
+import { Todo } from '../types/Todo';
+import React, { SetStateAction, Dispatch } from 'react';
+
+type FooterProps = {
+  todos: Todo[];
+  filterType: FilterType;
+  setFilter: Dispatch<SetStateAction<FilterType>>;
+};
+
+const filtersButtons = ['All', 'Active', 'Completed'];
```

Comment:
If you use enum to type filter and preparing filter section to scaling you can use this:
```
export enum FilterType {
  All = 'All',
  Active = 'Active',
  Completed = 'Completed',
}

const filterValues = Object.values(FilterType);
```
just make array of type/values from your enum type, and map in footer

Reply:
Oh, thats super simple, thank you!

Reply:
Also made these changes.